jsettlers.graphics/src/jsettlers/graphics/reader/translator/SettlerTranslator.java

```diff
@@ -44,9 +44,22 @@ public HeaderType getHeaderType() {
 	}
 
 	@Override
-	public short readUntransparentColor(ByteReader reader)
-			throws IOException {
-		return (short) ((reader.read16() << 1) | 0x01);
+	public short readUntransparentColor(ByteReader reader) throws IOException {
+		int read = reader.read16();
+
+		int R5 = (int)((read & 0xF800) >> 11);
```

Comment:
moved to Color as static https://github.com/jsettlers/settlers-remake/commit/6f07449288aa3ba3ec54fe2fbef31f710d17980d